sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -82,11 +88,18 @@ def create_key(self, name, key_type, **kwargs):
         enabled = kwargs.pop("enabled", None)
         not_before = kwargs.pop("not_before", None)
         expires_on = kwargs.pop("expires_on", None)
-        if enabled is not None or not_before is not None or expires_on is not None:
-            attributes = self._models.KeyAttributes(enabled=enabled, not_before=not_before, expires=expires_on)
+        exportable = kwargs.pop("exportable", None)
+
+        if enabled is not None or not_before is not None or expires_on is not None or exportable is not None:
```

Comment:
Suggestion: refactor this into a method you can reuse elsewhere, if indeed it checked elsewhere. In .NET I have 2 or 3 places I need to check this, and keeping each location up to date is risky. Instead, refactoring into a method you can use on the object to de/serialize consolidates the logic.